woudc_data_registry/models.py

```diff
@@ -1163,12 +1250,113 @@ def sync(ctx):
         Station,
         Instrument,
         Deployment,
-        DataRecord
+        DataRecord,
     ]
+    contribution_models = list()
+
+    """function to form all the contributions from the other models lists"""
+    def build_contributions(contribution_models, project_models,
+                            station_models, instrument_models, country_models,
+                            deployment_models, contributor_models):
+
+        """turn lists holding essential information into dictionaries with key
```

Comment:
Heads up that """ is used for the "docstring" at the start of a function definition, not for general comments.
Comments in Python start with `#`. Unlike other languages there's no multiline comments (e.g. `/* */`) so people just put a bunch of `#` comments together in a block.